buildbot/master/files/config/master.cfg

```diff
@@ -231,6 +232,16 @@ arm64_factory = create_servo_factory([
     steps.Compile(command=["./mach", "build", "--rel", "--target=aarch64-unknown-linux-gnu"], env=arm64_compile_env),
 ])
 
+windows_compile_env = dict({'PATH': r'C:\msys64\mingw64\bin;C:\msys64\usr\bin\C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files\Amazon\cfn-bootstrap',
+                            'MSYSTEM': 'MINGW64',
+                            'MSYS': 'winsymlinks=lnk'},
+                           **common_test_env)
```

Comment:
I'm skipping them because we only have one build type, at least for now, and I'm trying to get something that works similarly to the current AppVeyor config (https://github.com/servo/servo/blob/master/appveyor.yml).

I really wish that we could use ccache, but it doesn't work on Windows due to known build problems with the components we pick up from Firefox. I already spent a bunch of time trying to get it working (and failing) on AppVeyor and locally.